webserver.h

```diff
@@ -43,6 +42,61 @@ class WebServer
 
     void Start()
     {
+        // Define the `/api/sockets` endpoint
+        CROW_ROUTE(_crowApp, "/api/sockets")
```

Comment:
Is there a specific reason to not use the "`to_json()` in serialization.h" approach when it comes to the socket endpoints?

Reply:
Now I do!